doc/developers/maintainer.rst

```diff
@@ -1,14 +1,31 @@
 Maintainer / core-developer information
 ========================================
 
+A major release, incrementing the minor version, or a big fix release
+incrementing the patch version. Our convention is that we release one or more
+release candidates (0.XXrcN) before releasing the final distributions. Creating
+the first RC involves creating the release branch (0.99.X for instance)
+directly on the main repo. The rest of the changes should be done as a PR to
+the master branch, if it's possible, or done through a PR to the release
+branch, for the other maintainers to review before the release. Note that all
+development for the minor releases and the major release of for instance 0.99,
+happens under the branch called ``0.99.X``, where ``X`` is a literal, and
+independent of minor bug fix releases. Each release candidate, major, or minor
+release is a tag under the same branch. This is further explained under
+`Preparing a release PR<#Preparing-a-release-PR>`_. We follow the `PEP101
```

Comment:
shame on me....
line 15 should be
```
:ref: `preparing-a-release-pr`. We follow the `PEP101`
```
Then a label should be added before the section:
```
.. _preparing-a-release-pr:
```
:(

Reply:
no worries @cmarmo, I'm actually quite happy you figured it out. I'm not the most fluent when it comes to ReST as you can see :P